dapp-example/tests/support/bootstrap.js

```diff
@@ -29,6 +29,7 @@ if (supportedEnvs.includes(network.toLowerCase())) {
 
 client.setOperator(process.env.OPERATOR_ID_MAIN, process.env.OPERATOR_KEY_MAIN);
 console.log(`SDK setup for ${JSON.stringify(network)} for account: ${process.env.OPERATOR_ID_MAIN}`);
+console.log('Debugging');
```

Comment:
Lol, how does this fix it?

Reply:
Also we should fix main first since more PRs are going there and can chrery pick the final fix over once confirmed.

Reply:
It's not fixing it, I'm just testing... and the ci was completed successfully :confused:

Reply:
Marked the PR as a draft till debugging it.